app/src/controllers/userController.ts

```diff
@@ -0,0 +1,127 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+/**
+ * CRUD-Controller for user models
+ */
+export class UserController {
+
+    /**
+     * lists all available users
+     * @param  {Request} req - express request object
+     * @param  {Response} res — express response object
+     * @return json object with statusCode and list of users
+     */
+    public async index(req: Request, res: Response) {
+        try {
+            const users = await User.find();
+            res.json({
+                statusCode: 200,
+                data: users
+            });
+        } catch(err) {
+            res.json({
+                statusCode: 400,
+                errorMessage: err.message || err.toString()
+            });
+        }
+    }
+
+    /**
+     * creates a new user instance
+     * @param  {Request} req - express request object with updated user details in body
+     * @param  {Response} res — express response object
+     * @return json object with statusCode and and created user
+     */
+    public async create(req: Request, res: Response) {
```

Comment:
you need to specify return object format in all methods

Reply:
or type `void`

Reply:
You mean declaring it like: create(req: Request, res: Response): void 
I guess in this case void would be the most appropriate for all the methods?
or in the docblock like: @return {TYPE} json
Here I'm not sure what type that would be?